src/helpers/mapProps.ts

```diff
@@ -0,0 +1,26 @@
+import { StyleSheet } from 'react-native';
+import { MapPropsFunction } from './format';
+
+/**
+ * Preserve props that would cause an element to not be visible while stripping
+ * the rest
+ */
+export const mapVisibilityRelatedProps: MapPropsFunction = ({
+  accessibilityElementsHidden,
+  accessibilityViewIsModal,
+  importantForAccessibility,
+  style,
+}) => {
+  const styles = StyleSheet.flatten(style as any) ?? {};
+
+  return {
+    ...(accessibilityElementsHidden
```

Comment:
Pls log values for other props that might help user navigate the element tree:
* testID
* nativeID
* accessibilityLabel
* accessbilityLabeledBy
* accessibilityRole
* accessbilityHint
* accessibilityState
* accessibilityValue
* placeholder
* value
* defaultValue

Reply:
It terms of actual code we could have a list of allowed props to pass in:

```
const propsToDisplay = ['testID', 'nativeID', ... ];
```

For case when we would want to make some extra processing, like `accessibilityState`/`accessibilityValue` (remove `undefined` entries), we could add manual made `if` statements.

Reply:
Updated to map this logic.